### DISCUSSION TITLE
멀티플레이 간편링크에 UI Controller 와 캐릭터가 나타나지 않습니다.

### DISCUSSION THREAD

**Discussion by peterkangdev** [naverz/zepeto-studio-kor#298] [5. Q&A]
유니티 플레이를 할때는 UI Controller 와 캐릭터가 잘 나옵니다. 
그러나 QR 로 들어가면 나타나지 않습니다. 
이유를 아시는 분이 있으시면 설명 좀 해 주세요. 

혹시 아래와 관련이 있는지 모르겠네요..

<img width="825" alt="스크린샷 2022-01-30 오전 12 44 23" src="https://user-images.githubusercontent.com/30133892/151667272-18967dfd-a4c6-4895-997b-e7c2aa1f353e.png">

**Comment by runasy-koonta** [comment]
해당 오류는 Typescript가 컴파일 될 때마다 지속적으로 발생하더라구요. 아마 이번 문제와는 관련이 없을 것 같습니다.

캐릭터를 생성해주는 코드를 공유해주시면 문제 파악에 도움이 될 것 같습니다!
